### PR TITLE
chore(docs): rebuild table from all-tokens json

### LINKED ISSUE #68: Update documentation to add descriptions to tokens documentation
Improve the design tokens table to include token descriptions.

Add color swatches, similar to the colors page - see [Org PR](https://github.com/patternfly/patternfly-org/pull/4110) for updated implementation and [Core workspace](https://pf6.patternfly.org/components/alert#css-variables) examples as well

### PR DESCRIPTION
**Pull request by evwilkin**
Preview:  [Preview link](http://scary-brothers.surge.sh/tokens/all-patternfly-tokens)

Closes #68 

This PR implements new tokens tables which:
- display tokens based on their layer (semantic, base, palette, charts)
- displays default & dark theme values for all tokens
- displays token chains for both default & dark theme values
- pulls in new token descriptions and displays them (currently only applicable to semantic tokens)

README has been updated to explain commands to execute this code, but in summary the data needed for the table is pulled when `yarn build` or `yarn build:scss` is run (they trigger the same code), as two new json files containing all the token data displayed in the table are created by Style Dictionary in the same process as the scss files creation.

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]
Is there anything in the README which described how the tokens table was generated in the docs, which should now be updated?

**Review by srambach — COMMENTED** [review]
I'm not sure of the process for generating the docs - these token files are out of date so they really shouldn't be checked in. And maybe the docs need to be updated? +1 to @nicolethoen's comment that we need to document the process.

My only other comment is that the menu to filter the selection is a little odd with checkbox and also a check on hover. But I'm fine with refining that later.

image

**Comment by evwilkin** [comment]
Thanks @srambach for the feedback - I'll take a look at addressing your comments, not quite sure what I did to get those double checkmarks so thanks for catching that 🤔

**Review by srambach — APPROVED** [review]
A couple of comments, but they don't need to hold things up if you want to get this in.

File: packages/module/patternfly-docs/content/tokensTable.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  aspect-ratio: 1 / 1;
+  border-radius: 50%;
+  border: var(--pf-t--global--border--width--regular) solid var(--pf-t--global--background--color--inverse--default);
+  box-shadow: var(--pf-t--global--box-shadow--sm);
+}
+
+@media (min-width: 1200px) {
+  .tokens-table-outer-wrapper {
+    width: calc(92vw - var(--pf-v6-c-page__sidebar--Width));
+    max-width: calc(92vw - var(--pf-v6-c-page__sidebar--Width));
```
[the remark this answers is not in the document]
Comment (evwilkin) [reply packages/module/patternfly-docs/content/tokensTable.css:13]: Thanks for calling this out - our docs limits the width of the wrapper element that this content loads into, that value worked well when testing to increase the width of the table (to make it less squished unnecessarily) but yes we should swap it out for something more dynamic.  Opening a follow-up issue to review this suggestion 👏

File: packages/module/patternfly-docs/content/tokensTable.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.ws-token-swatch {
+  height: 1em;
+  display: inline-block;
+  aspect-ratio: 1 / 1;
+  border-radius: 50%;
```
[the remark this answers is not in the document]
Comment (evwilkin) [reply packages/module/patternfly-docs/content/tokensTable.css:5]: Thanks!  Opening a follow-up issue to swap this value out 👍

**Pull request merged**